src/Symbolica.Extensions.Configuration.FSharp/BindResult.fs

```diff
@@ -6,28 +6,61 @@ type BindResult<'a> =
 
 module BindResult =
 
+    /// <summary>
+    /// Applies the <see cref="BindResult" /> <paramref name="a" /> to the function <paramref name="f" />.
+    /// </summary>
+    /// <remarks>
+    /// This applicative instance accumulates errors.
+    /// If both <paramref name="f" /> and <paramref name="a" /> are <c>Success</c> then the outcome will be <c>Success</c>;
+    /// otherwise the outcome will be a <c>Failure</c> containing all errors.
+    /// </remarks>
+    /// <param name="f">The function to which the value will be applied.</param>
+    /// <param name="a">The value to be applied to the function.</param>
+    /// <returns>A <see cref="BindResult" /> containing the result of applying the value to the function.</returns>
     let apply f a : BindResult<'b> =
         match f, a with
         | Failure e1, Failure e2 -> Failure(List.append e1 e2)
         | Failure e1, Success _ -> e1 |> Failure
         | Success _, Failure e2 -> e2 |> Failure
         | Success f, Success a -> a |> f |> Success
 
+    /// <summary>Monadic bind for a <see cref="BindResult" />.</summary>
+    /// <remarks>
+    /// If the input <see cref="BindResult" /> is <c>Success</c> then the <paramref name="f" /> will be evaluated with
```

Comment:
You tend you use a construct like `the function <paramref name="f" />` in most places which reads slightly nicer than just `the <paramref name="f" />`.

Reply:
Yeah this seems like a typo here. Good spot.